packages/core/test/editable-text/editableTextTests.tsx

```diff
@@ -121,20 +122,87 @@ describe("<EditableText>", () => {
             assert.isTrue(confirmSpy.notCalled, "onConfirm called");
         });
 
-        it("calls onConfirm when cmd+enter or ctrl+enter is pressed", () => {
+        it("calls onConfirm when cmd+, ctrl+, shift+, or alt+ enter is pressed", () => {
             const confirmSpy = sinon.spy();
-            const wrapper = shallow(<EditableText isEditing={true} onConfirm={confirmSpy} multiline />);
-            wrapper.find("textarea")
-                .simulate("change", { target: { value: "control" } })
-                .simulate("keydown", { ctrlKey: true, which: Keys.ENTER });
+            const wrapper = mount(<EditableText isEditing={true} onConfirm={confirmSpy} multiline />);
+            simulateHelper(wrapper, "control", { ctrlKey: true, which: Keys.ENTER });
             wrapper.setState({ isEditing: true });
-            wrapper.find("textarea")
-                .simulate("change", { target: { value: "meta" } })
-                .simulate("keydown", { metaKey: true, which: Keys.ENTER });
+            simulateHelper(wrapper, "meta", { metaKey: true, which: Keys.ENTER });
+            wrapper.setState({ isEditing: true });
+            simulateHelper(wrapper, "shift", {
+                preventDefault: (): void => undefined,
+                shiftKey: true,
+                which: Keys.ENTER,
+            });
+            wrapper.setState({ isEditing: true });
+            simulateHelper(wrapper, "alt", {
+                altKey: true,
+                preventDefault: (): void => undefined,
+                which: Keys.ENTER,
+            });
             assert.isFalse(wrapper.state("isEditing"));
-            assert.isTrue(confirmSpy.calledTwice, "onConfirm not called twice");
+            assert.strictEqual(confirmSpy.callCount, 4, "onConfirm not called four times");
```

Comment:
@giladgray @cyu06 @themadcreator @llorca etc - I think we should align on using the `expect` API or the `assert` API across all tests, not both. I prefer `expect`, because those expressions read like plain English: `expect(blah).to.be.true`